<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="12155" started-at="2019-01-21T06:47:52Z" finished-at="2019-01-21T06:48:04Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="12155" started-at="2019-01-21T06:47:52Z" finished-at="2019-01-21T06:48:04Z">
      <class name="JavaTest.Fail">
        <test-method status="FAIL" signature="test_UsingHardAssertion()[pri:0, instance:JavaTest.Fail@28d25987]" name="test_UsingHardAssertion" duration-ms="3" started-at="2019-01-21T06:48:03Z" finished-at="2019-01-21T06:48:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HardAssertion] but found [HardAsser]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HardAssertion] but found [HardAsser]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at JavaTest.Fail.test_UsingHardAssertion(Fail.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_UsingHardAssertion -->
      </class> <!-- JavaTest.Fail -->
      <class name="JavaTest.Pass">
        <test-method status="PASS" signature="loginsetup()[pri:0, instance:JavaTest.Pass@59fa1d9b]" name="loginsetup" is-config="true" duration-ms="7014" started-at="2019-01-21T06:47:52Z" finished-at="2019-01-21T06:47:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginsetup -->
        <test-method status="PASS" signature="test()[pri:0, instance:JavaTest.Pass@59fa1d9b]" name="test" duration-ms="4295" started-at="2019-01-21T06:47:59Z" finished-at="2019-01-21T06:48:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="aftertest()[pri:0, instance:JavaTest.Pass@59fa1d9b]" name="aftertest" is-config="true" duration-ms="820" started-at="2019-01-21T06:48:03Z" finished-at="2019-01-21T06:48:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- JavaTest.Pass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
